features/cells/src/main/java/com/wire/android/feature/cells/ui/filter/FilterBottomSheet.kt

```diff
@@ -66,6 +67,7 @@ fun FilterBottomSheet(
     onApply: (Set<String>) -> Unit,
     onClearAll: () -> Unit,
     onDismiss: () -> Unit,
+    onShow: () -> Unit = {},
```

Comment:
`onShow` suggests (at least to me) that it triggers each time the bottom sheet is opened 😄 Maybe something like `onRefreshContent` would be better?

Reply:
As we discussed on 1:1, it would stack up for a regular flow, but when using `snapshotFlow` it does not - it's not emitting new value until old one is not finished collecting, probably because "State change events are conflated" (it's from the `snapshotFlow` docs), so current solution should already work fine.